### PR TITLE
Muki lab02 converter

### PR DESCRIPTION
**Pull request by Muki-has-ideas**
Lab 02 done up through version 2.

### CODE REVIEW

**Review by brucestull — CHANGES_REQUESTED** [review]
[Lab 02: Unit Converter](https://github.com/PdxCodeGuild/class_HB2/blob/main/1%20Python/labs/02%20Unit%20Converter)

**Needs Work**

Comments:
* User input of `72` `in` results in no output. Expected output is some value in `meters`.
    * Sample input and output:
    ```
    PS C:\Users\Bruce\Programming\class_HB2\code\muki> git status
    On branch muki-lab02-converter
    Your branch is up to date with 'origin/muki-lab02-converter'.

    nothing to commit, working tree clean
    PS C:\Users\Bruce\Programming\class_HB2\code\muki>
    PS C:\Users\Bruce\Programming\class_HB2\code\muki> python .\lab02.py

    Enter the length that you would like to convert to meters:      72
    Enter the units: (km, mi, ft, in, mm, yd:       in
    PS C:\Users\Bruce\Programming\class_HB2\code\muki>
    ```
* Math operators should have single-space before and after. See [PEP8 style guide](https://peps.python.org/pep-0008/#other-recommendations)
* Great use of float so output can be non-integer.

File: code/muki/lab02.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+m_yd = 1 / 0.9144
+m_m = 1
+in_m = 0.0254
+m_in = 1 / 0.0254
+
+dist = input('\nEnter the length that you would like to convert to meters:\t')
+dist_unit = input('Enter the units: (km, mi, ft, in, mm, yd:\t')
+
+
+if dist_unit.lower() == 'km':
```
Comment (brucestull) [inline code/muki/lab02.py:82]: * Math operators should be surrounded by single-spaces for readability.
    * `variable_one*variable_two` should be `variable_one * variable_two`

**Review by brucestull — APPROVED** [review]
[Lab 02: Unit Converter](https://github.com/PdxCodeGuild/class_HB2/blob/main/1%20Python/labs/02%20Unit%20Converter)

**Completed**

Comments:
* Lab performs requirements
* In future labs, be aware of PEP8 Style guide. Ensure proper whitespace to enhance code readability.

**Comment by Muki-has-ideas** [comment]
thanks Bruce

**Pull request merged**